### ISSUE TITLE
Get Model associations camelCased

### ISSUE THREAD

**Issue by Globegitter** [balderdashy/sails#2679]
When working with Ember/JSONApi it turns out it is quite hard to get associations in camelCase as these standards expect.

So if I have a model `researchprojectactivities` Ember/JSON Api expects `researchProjectActivities` wherever it is being used. It is not too hard to camelCase when you just get the model, because you can camelCase the `globalId`, but when you work with relationships you now have to do workarounds like `toCamelCase(sails.models[assoc.collection].globalId`, so work with the global sails object.

I have looked into the `orm` hook and I can see in `build-orm.js` it is already lowercased. And I see there is a `normalize-model.js` but I could not find where exactly the association gets lowercased with a bit of debugging.

Maybe as there is a `pluralize` setting we could add a `camelCase: true/false` setting. I would be more than happy to make a PR but maybe @sgress454 or @mikermcneil can point me to where exactly this is done? Would save me some more debugging time :P

**Comment by sgress454** [comment]
Actually @particlebanana would probably know better than either Mike or I...

**Comment by tjwebb** [comment]
@sgress454 @particlebanana any ideas on this?

**Comment by particlebanana** [comment]
Not sure I follow, sorry not at all familiar with Ember. You can name model attributes whatever you want. Are you talking about the globals that get generated?

**Comment by sailsbot** [comment]
Thanks for posting, @Globegitter.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
